source/Octopus.Tentacle.Client/Scripts/ScriptExecutionOrchestrator.cs

```diff
@@ -477,7 +477,14 @@ void CancelAfter()
                         }
 
                         var abandonTask = abandonCancellationTokenSource.Token.AsTask<ScriptStatusResponseV2>();
-                        await (await Task.WhenAny(actionTask, abandonTask).ConfigureAwait(false)).ConfigureAwait(false);
+                        var task = await Task.WhenAny(actionTask, abandonTask);
+
+                        if (task == abandonTask)
```

Comment:
Not that I personally care, but I thought I'd point out that the "if" statement here is the reverse of what we do elsewhere.

I.e. it would normally be `if (task != actionTask)`